test/parallel/test-cli-eval.js

```diff
@@ -36,12 +38,14 @@ child.exec(nodejs + ' --eval "console.error(42)"',
 
   child.exec(cmd + '42',
              function(err, stdout, stderr) {
+               assert.strictEqual(typeof err, 'object');
```

Comment:
This is OK, but is probably better like this:

```js
assert(err instanceof Error);
```

Even better if the type of error is known. For example, if it's expected to be a `RangeError`, then `assert.(err instanceof RangeError)` etc.